main.go

```diff
@@ -6,7 +6,7 @@ import (
 	"path"
 	"strconv"
 
-	"github.com/codegangsta/cli"
+	"github.com/docker/machine/cli"
```

Comment:
I have made a change to some of the command / flag parsing logic to address a bug in the library and this is intended as a temporary step.  Ideally I'd like to submit the fix upstream and get it merged in.